### PR TITLE
feat(SwAvatar): add dynamic background and color groups

### PR DESCRIPTION
**Pull request by YossiSaadi**

### CODE REVIEW

File: src/components/Avatar/SwAvatar.vue [hunk 1]
```diff
@@ -16,13 +16,15 @@ const props = defineProps({
...
+  border: { type: Boolean, default: true },
 });
 
 const letter = computed(() => props.text.charAt(0));
+const colorGroup = computed(() => (letter.value.charCodeAt(0) % 5) + 1);
 </script>
 
 <template>
-  <div :class="['wrapper', [shape], [size]]">
+  <div :class="['wrapper', [`group${colorGroup}`], { shape, size, border }]">
```
Comment (talkor) [inline src/components/Avatar/SwAvatar.vue:27]: Where is the `group${colorGroup}` class?

Comment (talkor) [reply src/components/Avatar/SwAvatar.vue]: I think we should

File: src/components/Avatar/SwAvatar.vue [hunk 2]
```diff
@@ -24,7 +24,8 @@ const colorGroup = computed(() => (letter.value.charCodeAt(0) % 5) + 1);
 </script>
 
 <template>
-  <div :class="['wrapper', [`group${colorGroup}`], { shape, size, border }]">
+  <div :class="['wrapper', [shape], [size]]">
+  <div :class="['wrapper', `group${colorGroup}`, shape, size, border]">
     <img class="image" v-if="src" :src="src" :alt="text" />
     <span v-else class="capitalize">{{ letter }}</span>
   </div>
```
[the remark this answers is not in the document]
Comment (YossiSaadi) [reply src/components/Avatar/SwAvatar.vue:30]: yep, fixed, it's a weird bug on GH's editor

File: src/components/Avatar/SwAvatar.vue [hunk 3]
```diff
@@ -33,27 +35,28 @@ const letter = computed(() => props.text.charAt(0));
...
 }
 
-.image {
+.wrapper.image {
   width: 100%;
   object-fit: cover;
 }
 
-.capitalize {
+.wrapper.capitalize {
```
Comment (talkor) [inline src/components/Avatar/SwAvatar.vue:47]: We need postcss nested - we have a task for BH

**Review by talkor — APPROVED** [review]

**Pull request merged**